kube-ui/kube-ui-rc.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: kube-ui-v1
+  namespace: default
+  labels:
+    k8s-app: kube-ui
+    version: v1
+    kubernetes.io/cluster-service: "true"
```

Comment:
I'm not sure whether having these config files makes sense here. Since they're running as cluster addons, I imagine these will need to remain in the k8s repo even after www/... is removed.